core/src/test/java/org/elasticsearch/http/netty/NettyHttpChannelTests.java

```diff
@@ -414,5 +460,68 @@ public BytesReference content() {
         public RestStatus status() {
             return RestStatus.OK;
         }
+
+    }
+
+    private static class TestCircuitBreaker implements CircuitBreaker {
```

Comment:
OK for now, but I guaranteed someone will use this and be surprised. But sure, we can wait.